### ISSUE TITLE
Please update to v38.

### ISSUE THREAD

**Issue by Likon84** [ricardoquesada/Spidermonkey#35]
Please update to v38 current v34 is a bit outdated...

**Comment by ricardoquesada** [comment]
yeah... no time for that at the moment, but if you send me a pull request I'll merge it. thanks.

**Comment by Likon84** [comment]
maybe you can find some time for it? I think it require c++ code changes and I'm a JS developer... I know there are performance upgrade in the newest version... it will speed up whole cocos
